package.json

```diff
@@ -12,11 +13,22 @@
   ],
   "license": "BSD-3-Clause",
   "scripts": {
-    "build": "tsc && webpack -p",
-    "lint": "eslint -c eslint.json src/**/*.ts",
-    "clean": "rimraf dist lib",
-    "prepack": "npm run clean && npm run build",
-    "watch": "tsc -w --listEmittedFiles"
+    "build": "jlpm run build:lib && jlpm run build:labextension:dev",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "build:prod": "jlpm run build:lib && jlpm run build:labextension",
+    "clean": "jlpm run clean:lib",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+    "clean:labextension": "rimraf pyviz_comms/labextension",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "eslint": "eslint . --ext .ts,.tsx --fix",
+    "eslint:check": "eslint . --ext .ts,.tsx",
+    "install:extension": "jupyter labextension develop --overwrite .",
+    "prepare": "jlpm run clean && jlpm run build:prod",
+    "watch": "run-p watch:src watch:labextension",
+    "watch:labextension": "jupyter labextension watch .",
+    "watch:src": "tsc -w"
```

Comment:
@mattpap How much of the old scripts was custom? Do we have to keep a `build:nbextension` script or similar?